### ISSUE TITLE
TF2.0 SavedModel with feature processing inference code 

### ISSUE THREAD

**Issue by 17patelumang** [tensorflow/tensorflow#43350]
**System information** 

- OS Platform and Distribution : macOS Catalina 10.15.3

- TensorFlow installed from : binary

- TensorFlow version : 2.2.0
- Python version: 3.7.3

**Describe the current behavior**

Hi ,

I am looking for sample code to do inference in TF2.0 . We have savedModel in TF2.0 with feature processing as part of saved model.

In TF1.x we used to do inference as follows 

```
import tensorflow as tf

saved_model_path = "<path>""


with tf.Session(graph=tf.Graph()) as sess:
  metagraph = tf.saved_model.loader.load(sess, [tf.saved_model.tag_constants.SERVING], saved_model_path)
  prediction = sess.run(['loss/scores:0'], feed_dict={'<some place holder>':<input_list>})
  print(prediction)
```
what will be corresponding code in TF2.0  since it doesnot use session concept ?  
Note: I dont need to have  working code , please point me to the sample code somethere

**Comment by bhack** [comment]
Check https://www.tensorflow.org/guide/saved_model

**Comment by 17patelumang** [comment]
@bhack we checked  that but there is  no clear code, could you p lease share  sample code where we need to specify 'loss/scores:0' & feed_dict ?

**Comment by bhack** [comment]
I think the tutorial is very clear. You can interactively run and modify yourself at https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/guide/saved_model.ipynb.
If you want to improve this tutorial please open a documentationt type issue (not a bug as this) or a PR directly.

For support questions please use our Stackoverflow: https://stackoverflow.com/questions/tagged/tensorflow

Thanks

**Comment by 17patelumang** [comment]
@bhack thank you for reply. We read this documentation before but in this documentation there is no sample code for inference from savedModel format which has TF feature processing part of  it . Could you please provide sample code.

**Comment by bhack** [comment]
See this comment and the next at https://github.com/tensorflow/tensorflow/issues/31055#issuecomment-516205819

**Comment by 17patelumang** [comment]
@bhack thank you  for reply , appreciate it ! , but where to find the inference code ?

**Comment by bhack** [comment]
That example is about using preprocessing with saved model e.g. for TF serving. 
What is you use case?

**Comment by 17patelumang** [comment]
@bhack thank you for reply , we have savedModel with TF feature processing part of it in TF 2.x, however we are un able to find the sample inference code in TF2.x from savedModel which has TF feature processing part of it . In the above , we have TF 1.15.x  inference code , where can we refer for similar inference code in TF 2.x ?

**Comment by bhack** [comment]
Do you have a very very minimal but runnable TF2.x example or colab up to the save command?

Generally I think that this is a support question for: https://stackoverflow.com/questions/tagged/tensorflow

But if you can share something in TF 2.x we could see if there is something that could be improved in the docs.

**Comment by 17patelumang** [comment]
I already created https://stackoverflow.com/questions/63962865/tf2-0-savedmodel-with-feature-processing-inference-code but no reply.  We have savedModel in TF 2.x , dont have the code to save model . We are looking for way to do the inference in TF 2.x

**Comment by bhack** [comment]
Sorry but all the step in https://www.tensorflow.org/guide/saved_model starting from
```
loaded = tf.saved_model.load(mobilenet_save_path)
print(list(loaded.signatures.keys()))  # ["serving_default"]
```
Is not inference with savedModel?

**Comment by bhack** [comment]
If you are looking instead about how to embed a preprocessing function inside a SavedModel you can take a look at https://sayak.dev/tf.keras/preprocessing/2020/04/13/embedding-image-preprocessing-functions.html

**Comment by 17patelumang** [comment]
thank you
